packages/gatsby-theme-docs/src/templates/releases.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from 'gatsby';
+import { MDXRenderer } from 'gatsby-plugin-mdx';
+import { MDXProvider } from '@mdx-js/react';
+import {
+  Markdown,
+  Subtitle,
+  ContentNotifications,
+} from '@commercetools-docs/ui-kit';
+import LayoutContent from '../layouts/content';
+import {
+  SEO,
+  Link,
+  ThemeProvider,
+  CodeExample,
+  MultiCodeExample,
+} from '../components';
+import PlaceholderMarkdownComponents from '../overrides/markdown-components';
+
+// See https://mdxjs.com/getting-started#table-of-components
+const components = {
+  // UI components
+  p: Markdown.Paragraph,
+  // NOTE: we want to ensure that only one h1 exists on each page.
+  // Therefore, we map the markdown header elements starting from h2.
+  // The h1 header will be automatically rendered based on the page title.
+  h1: Markdown.withAnchorLink(Markdown.H2),
+  h2: Markdown.withAnchorLink(Markdown.H3),
+  h3: Markdown.withAnchorLink(Markdown.H4),
+  h4: Markdown.withAnchorLink(Markdown.H5),
+  h5: Markdown.withAnchorLink(Markdown.H6),
+  h6: Markdown.withAnchorLink(Markdown.H6),
+  thematicBreak: Markdown.ThematicBreak,
+  blockquote: Markdown.Blockquote,
+  ul: Markdown.Ul,
+  ol: Markdown.Ol,
+  li: Markdown.Li,
+  dl: Markdown.Dl,
+  dt: Markdown.Dt,
+  dd: Markdown.Dd,
+  table: Markdown.Table,
+  tr: Markdown.TableRow,
+  td: Markdown.TableCell,
+  th: Markdown.TableHeader,
+  code: Markdown.InlineCode,
+  inlineCode: Markdown.InlineCode,
+  em: Markdown.Em,
+  strong: Markdown.Strong,
+  delete: Markdown.Delete,
+  hr: Markdown.Hr,
+  pre: Markdown.CodeBlock,
+  // Custom component specific to the Gatsby theme
+  a: Link,
+
+  // Official react components to be used in MDX files
+  Subtitle,
```

Comment:
Subtitle should not be allowed in release notes  (Assuming this template is for the release note details page - on the top of the overview page we do need it though). 

If we omit it here using it in a release note will either break or just render a div (not sure, but both would be good)